tools/PI/DevHome.PI/ViewModels/BarWindowViewModel.cs

```diff
@@ -43,6 +39,9 @@ public partial class BarWindowViewModel : ObservableObject
     private readonly ObservableCollection<Button> _externalTools = [];
     private readonly SnapHelper _snapHelper;
 
+    [ObservableProperty]
```

Comment:
This isn't an observable property, correct? Just a private member?

Reply:
Good catch, I started it with an observable property, forgot to remove it.